### PR TITLE
Added save configuration button

### PR DESCRIPTION
**Pull request by jedouard98**
Added save configuration button based on offline discussion. Works the same as BuildSettingsWindow.
Currently the button saves all settings made across all tabs and is present in all of them.

Changed GUI:

image

image

image

image

image

### CODE REVIEW

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 1]
```diff
@@ -162,7 +186,7 @@ private void OnGuiVerifyBundleSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("AssetBundle URL", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.assetBundleUrl = EditorGUILayout.TextField(QuickDeployConfig.Config.assetBundleUrl,
+            _assetBundleUrl = EditorGUILayout.TextField(_assetBundleUrl,
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Nit: one line now

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 2]
```diff
@@ -233,11 +256,10 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.apkFileName =
-                EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName, GUILayout.MinWidth(FieldMinWidth));
+            _apkFileName = EditorGUILayout.TextField(_apkFileName, GUILayout.MinWidth(FieldMinWidth));
             if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
             {
-                QuickDeployConfig.Config.apkFileName = EditorUtility.SaveFilePanel("Choose file name and location", "",
+                _apkFileName = EditorUtility.SaveFilePanel("Choose file name and location", "",
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: One line

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 3]
```diff
@@ -40,16 +40,35 @@ public enum ToolBarSelectedButton
...
         private const string LoadingScreenErrorTitle = "Creating Loading Scene Error";
         private const string OkButtonText = "OK";
 
+        private string _assetBundleFileName;
+        private string _cloudStorageBucketName;
+        private string _cloudStorageFileName;
+        private string _cloudCredentialsFileName;
+
+        private string _assetBundleUrl;
+
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Delete newline

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 4]
```diff
@@ -40,16 +40,35 @@ public enum ToolBarSelectedButton
...
-        
+
         private const string LoadingScreenErrorTitle = "Creating Loading Scene Error";
         private const string OkButtonText = "OK";
 
+        private string _assetBundleFileName;
+        private string _cloudStorageBucketName;
+        private string _cloudStorageFileName;
+        private string _cloudCredentialsFileName;
+
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Delete newline

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 5]
```diff
@@ -40,16 +40,35 @@ public enum ToolBarSelectedButton
         private const int ButtonWidth = 200;
         private const int LongButtonWidth = 300;
         private const int ShortButtonWidth = 100;
-        
+
         private const string LoadingScreenErrorTitle = "Creating Loading Scene Error";
         private const string OkButtonText = "OK";
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs:46]: Add a comment:

// Local copy of the fields from QuickDeployConfig.Config to track unsaved changes.

**Review by thanwah — APPROVED** [review]
LGTM, thanks.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 5 above)
Comment (jedouard98) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs:46]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 4 above)
Comment (jedouard98) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 3 above)
Comment (jedouard98) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 1 above)
Comment (jedouard98) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 2 above)
Comment (jedouard98) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

**Pull request merged**